sparrow-py/pysrc/sparrow_py/expr.py

```diff
@@ -0,0 +1,209 @@
+"""Defines classes representing Kaskada expressions."""
+
+import sys
+from typing import Callable
+from typing import Tuple
+from typing import Union
+from typing import final
+
+from sparrow_py import ffi
+
+
+class Expr(object):
+    """A Kaskada expression."""
```

Comment:
Should improve the pydocs at some point based on existing docs, but this is a good start.